modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -34,13 +36,8 @@ Callback authCallback = new Callback<UserPrincipal>() {
 authService.login(options, authCallback);
 ----
 
-Once the browser returns to the app the result can be handled. In the activity provided to the
-`login` method override `onActivityResult`. This handler should always invoke
-`AuthService#handleAuthResponse`, providing the `Intent`. This will exchange the temporary tokens
-returned from `AuthService#login` for long-life tokens and will provide a `UserPrincipal` which can
-be used to access a users details. If this is not invoked you will not have access to the
-`UserPrincipal`.
-
+. Override `handleAuthResult` to handle the result from the browser
++
 [source,java]
 ----
 @Override
```

Comment:
A different code is being used to get the instance here.
`mobileCore.getInstance(AuthService.class)`
And in the previous one is
`MobileCore.getInstance().getService(AuthService.class)`
This is confusing, the former one is an instance of MobileCore but the latter is calling a static method.
Let's be consistent. I think this has to work as is so I think the correct one is the static method.